lib/generators/templates/simple_form_for/passwords/edit.html.erb

```diff
@@ -7,7 +7,12 @@
   <%= f.full_error :reset_password_token %>
 
   <div class="form-inputs">
-    <%= f.input :password, label: "New password", required: true, autofocus: true, hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length) %>
+    <%= f.input :password,
+                label: "New password",
+                required: true,
+                autofocus: true,
+                hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
+                input_html: { autocomplete: "new-password" } %>
     <%= f.input :password_confirmation, label: "Confirm your new password", required: true %>
```

Comment:
Several of the other password_confirmation fields in this PR were updated to include autofocus, should this one have been updated as well?

Reply:
I went to look at the original code and found this comment: https://github.com/plataformatec/devise/commit/6260c29a867b9a656f1e1557abe347a523178fab#r31284249

So yeah, it looks like we forgot. Are you willing to take a stab on this one?

Reply:
Sorry, but I don't think I'll have the time to take this on. I just wanted to call it out in case it was an oversight so that it could be fixed later.

Reply:
Sure thing, no problem. Thanks for pointing this out!